### DISCUSSION TITLE
DHCP lease time

### DISCUSSION THREAD

**Discussion by DavorJ** [strongswan/strongswan#1028] [Q&A] [answered]
I am using the [DHCP plugin](https://docs.strongswan.org/docs/5.9/plugins/dhcp.html) with identity lease. The DHCP server leases IPs for a duration of 12h. All works well: the road-warriors get their virtual IP from the DHCP server.

The problem is after 12h: the DNS server can't resolve the road-warrior name any more.

I am trying to debug/understand this. Could it be that the DHCP plugin is ignoring the lease time?

**Comment by tobiasbrunner** [comment, marked as answer]
> Could it be that the DHCP plugin is ignoring the lease time?

It is.

**Comment by DavorJ** [comment]
Any workaround or best practice?

**Reply by tobiasbrunner** [reply]
Increase the lease time on the server? Reconnect the clients earlier (e.g. via reauthentication)? Use static leases and fixed DNS entries?
